content/en/blog/_posts/2024-12-11-Kubernetes-v1-32-Release.md

```diff
@@ -0,0 +1,302 @@
+We've implemented QueueingHints in all plugins, and are moving ahead enabling this feature by default.
+
+This work was done as a part of [KEP #4247](https://github.com/kubernetes/enhancements/issues/4247), by [SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling).
+
+### Recover from volume expansion failure
+
+This feature moved to Beta in v1.32, allowing users to recover from volume expansion failure by retrying with a smaller size. This enhancement ensures that volume expansion is more resilient and reliable, reducing the risk of data loss or corruption during the process.
+
+This work was done as a part of [KEP #1790](https://github.com/kubernetes/enhancements/issues/1790), by [SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage).
+
+### Volume group snapshot
+
+This feature introduces a VolumeGroupSnapshot API to take a snapshot of multiple volumes together.
+Allows users to take a snapshot of multiple volumes together, ensuring data consistency across the volumes.
+
+This work was done as a part of [KEP #3476](https://github.com/kubernetes/enhancements/issues/3476), by [SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage).
+
+### Structured parameter support
+
+The core part of Dynamic Resource Allocation (DRA), the structured parameter support, got promoted to beta.
+Thanks to this feature, the kube-scheduler and Cluster Autoscaler can handle respectively simulate claim allocation themselves without relying on a third-party driver.
+
+This work was done as a part of [KEP #4381](https://github.com/kubernetes/enhancements/issues/4381), by WG Device Management (a cross functional team containing [SIG Node](https://github.com/kubernetes/community/tree/master/sig-node), [SIG Scheduling](https://github.com/kubernetes/community/tree/master/sig-scheduling) and [SIG Autoscaling](https://github.com/kubernetes/community/tree/master/sig-autoscaling)).
+
+## New features in Alpha
+
+_This is a selection of key improvements introduced as alpha features in the v1.32 release._
+    
+### Asynchronous preemption in the Kubernetes Scheduler
+
+Enhance the scheduling throughput when the preemption is involved, by making some expensive operations within the preemption asynchronously.
+This improvement is particularly beneficial in clusters with high pod churn or frequent scheduling failures, ensuring a more efficient and resilient scheduling process.
+
+The Kubernetes scheduler has been enhanced with Asynchronous Preemption, a feature that improves scheduling throughput by handling preemption operations asynchronously. Preemption ensures higher-priority pods get the resources they need by evicting lower-priority ones, but this process previously involved blocking operations like PostFilter execution, slowing down the scheduler. With this enhancement, such tasks are now processed in parallel, allowing the scheduler to continue scheduling other pods without delays.
```

Comment:
This is awesome, I would cut the previous paragraph and go with this one alone

Reply:
Fixed, just kept the reason why the improvement is beneficial at the end